manifests/overlays/authentication/deployment.yaml

```diff
@@ -15,6 +15,7 @@
       - --pass-access-token
       - '--openshift-delegate-urls={"/": {"resource": "services", "verb": "get", "name": "odh-dashboard"}}'
       - --skip-auth-regex=^/metrics
+      - --cookie-expire=24h0m0s
```

Comment:
https://github.com/openshift/oauth-proxy/issues/225

Reply:
Maybe we can include this in https://github.com/opendatahub-io/odh-dashboard/pull/823 cc @LaVLaS

Reply:
Yep. I'll add it to #823.  Is the value supposed to be `23h` or `24h` since the downstream hotfix value is `23h`

Reply:
It's gone now, we were going back and forth on the value @LaVLaS -- 23h is what Chris picked in the end.